holoviews/tests/operation/test_datashader.py

```diff
@@ -1169,6 +1171,24 @@ def test_rasterize_image_expand_default(self):
         output = img.data["z"].to_numpy()
         assert np.isnan(output).any()
 
+    def test_rasterize_apply_when_instance_with_line_width(self):
+        df = pd.DataFrame(
```

Comment:
I think we should reduce the size of this DataFrame. No need for it to be this big for a unit test.